qtpy/__init__.py

```diff
@@ -114,15 +114,17 @@ class PythonQtWarning(Warning):
 PYQT5 = True
 PYQT4 = PYSIDE = PYSIDE2 = False
 
-
-if 'PyQt5' in sys.modules:
-    API = initial_api if initial_api in PYQT5_API else 'pyqt5'
-elif 'PySide2' in sys.modules:
-    API = initial_api if initial_api in PYSIDE2_API else 'pyside2'
-elif 'PyQt4' in sys.modules:
-    API = initial_api if initial_api in PYQT4_API else 'pyqt4'
-elif 'PySide' in sys.modules:
-    API = initial_api if initial_api in PYSIDE_API else 'pyside'
+# When `FORCE_QT_API` is set to "1", we disregard
+# any previously imported python bindings.
+if os.environ.get('FORCE_QT_API') == '1':
```

Comment:
Let's change this line to

    if os.environ.get('FORCE_QT_API') is not None:

to avoid relying on a predetermined value of `1`, which I find a bit odd.

In this case, if `FORCE_QT_API` is set to any value, then the code inside this `if` won't take effect.

Reply:
Updated.

A value of 1 or 0 is pretty common for boolean environment variables, but it's not a technical feature of the shell.  A value like that can interface nicely in different languages; like javascript where `true == "1"` actually evaluates to True!